shared_task_code/ecs_task_queue_consumer.py

```diff
@@ -0,0 +1,91 @@
+import os
+import aioboto3
+import boto3
+from utils.lambda_decorators import ssm_parameters, async_handler
+from utils.json_serialisation import dumps
+from json import loads
+from datetime import datetime
+import subprocess
+from shared_task_code.task_queue_consumer import TaskQueueConsumer
+
+
+class ECSTaskQueueConsumer(TaskQueueConsumer):

```

Comment:
I thought we were going to add an abstract method on the consumer to implement the validation of the task parameters. ?